androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -48,6 +49,9 @@ public class TwaLauncher {
 
     private static final int DEFAULT_SESSION_ID = 96375;
 
+    private static final String EXTRA_STARTUP_UPTIME_MILLIS =
+            "android.support.customtabs.trusted.STARTUP_UPTIME_MILLIS";
```

Comment:
nit: I'd probably not have this be `android.support.customtabs` as it's not in (and not going to be in) AndroidX (which used to be the Android Support Library). Since it's only going to be read by Chromium, `org.chromium.chrome.browser.customtabs.trusted.STARTUP_TIME_MILLIS` or something should be good.

Reply:
Thank you for the review. Applied to the new commit.